source/ShipInfoDisplay.cpp

```diff
@@ -253,6 +254,38 @@ void ShipInfoDisplay::UpdateAttributes(const Ship &ship, const Depreciation &dep
 		attributesHeight += 20;
 	}
 	
+	// Miscellaneous stats
+	if(attributes.Get("scan interference"))
+	{
+		attributeLabels.push_back("chance to block scan:");
```

Comment:
"block scan" sounds awkward here. How about "Chance to evade scan"

Reply:
Really what you are doing is spoofing the scan. You aren't blocking it or evading it. It happens to you, and returns false results. Spoof might be a bit to goofy though. Perhaps deceive?

Reply:
If you don't mind my novice-level necromancy, perhaps consider "chance to fool scanners:"?